tests/constants/admincli/cmd.go

```diff
@@ -25,4 +25,28 @@ const (
 
 	// ListVolumes referring to vmdkops_admin volume ls
 	ListVolumes = vmdkopsAdminVolume + "ls "
+
+	// vmdkopsCLIVolumegroup Volume group CLI commands
+	vmdkopsCLIVolumegroup = vmdkopsAdmin + "vmgroup"
```

Comment:
1. Volumegroup should be VmGroup. Same for other constants below.
2. Let's have consistent prefix, either "vmdkopsAdmin" or "vmdkopsCLI", or simply without prefix since there's no naming conflicts in this const type.